openapi/v2/core-data.yaml

```diff
@@ -278,6 +262,7 @@ components:
               description: "A string representation of the reading's value"
               type: string
       required:
+        - valueType
```

Comment:
Per this [ conversation  ](https://github.com/edgexfoundry/go-mod-core-contracts/pull/264#discussion_r468700735)we will need valueType moved to BaseReading where it will be required

Reply:
Updated to move `valueType` to `BaseReading`.